content/documentation.rst

```diff
@@ -91,7 +142,11 @@ documenting C and C++ is also supported.
 Sphinx supports several output formats directly, such as HTML, LaTeX, and ePub,
 and supports PDF output via either LaTeX or the external rst2pdf tool.
 
-For us, rarrative documentation is delivered in the reStructuredText (.rst)
+Due to the ability to output to several formats, keep in mind how the output
+will look like. For example, raw HTML is discouraged as this will affect the
```

Comment:
There is a section at the bottom of the documentation page  **Mock ups and designs for the website** which already mentions that you can use html etc
This section is just for documentation only